### ISSUE TITLE
Getting traceback error from /usr/local/bin/cot

### ISSUE THREAD

**Issue by drq883** [coteditor/CotEditor#877]
I'm getting error trying to run /usr/local/bin/cot
  (symlinked to /Applications/CotEditor.app/Contents/SharedSupport/bin/cot)

Error is:

```
% /usr/local/bin/cot
Traceback (most recent call last):
  File "/usr/local/bin/cot", line 360, in <module>
    main(args, stdin)
  File "/usr/local/bin/cot", line 269, in main
    client_name = system.tell('name of first process whose frontmost is true')
  File "/usr/local/bin/cot", line 120, in tell
    return run_osascript(script, is_async)
  File "/usr/local/bin/cot", line 92, in run_osascript
    raise CalledProcessError(p.returncode, script)
subprocess.CalledProcessError: Command 'tell app "System Events" to name of first process whose frontmost is true' returned non-zero exit status 1
```
Once I start up CotEditor from spotlight, it works.

My version of CotEditor is 3.6.7 (301)

**Comment by 1024jp** [comment]
Thank you for the report. However, I cannot reproduce your error.
Could you give me the following info?:

- macOS version.
- Name & version of the terminal application you use.
- Is there any difference when you invoke `/usr/local/bin/cot` and just `cot`

**Comment by drq883** [comment]
Try as I might, I can't recreated the error on the Mac at home or at work. I'm going to close it. I don't know what was the matter. Thanks for looking at it.

**Comment by 1024jp** [comment]
Oh, ok. Thanks anyway.
Feel free to open an issue again when you run across it again.

**Comment by elkssing** [comment]
I'm seeing this issue.  It doesn't matter whether CotEditor is already open and it doesn't matter how I start it.  I'm using Terminal Version 2.9.1 (421.1), latest version as of today.  Here's the issue template with more:

## Environment

- CotEditor: 3.6.9 (305)
- System: macOS Version 10.14.2 (Build 18C54)
- Language: English


## Short Description

traceback from cot command line


## Steps to Reproduce the Issue
```
$ ls -l /usr/local/bin/cot
lrwxr-xr-x  1 sge  staff  58 Dec 28 05:48 /usr/local/bin/cot -> /Applications/CotEditor.app/Contents/SharedSupport/bin/cot
$ cot .bashrc
Traceback (most recent call last):
  File "/usr/local/bin/cot", line 360, in <module>
    main(args, stdin)
  File "/usr/local/bin/cot", line 269, in main
    client_name = system.tell('name of first process whose frontmost is true')
  File "/usr/local/bin/cot", line 120, in tell
    return run_osascript(script, is_async)
  File "/usr/local/bin/cot", line 92, in run_osascript
    raise CalledProcessError(p.returncode, script)
subprocess.CalledProcessError: Command 'tell app "System Events" to name of first process whose frontmost is true' returned non-zero exit status 1
$ cot
Traceback (most recent call last):
  File "/usr/local/bin/cot", line 360, in <module>
    main(args, stdin)
  File "/usr/local/bin/cot", line 269, in main
    client_name = system.tell('name of first process whose frontmost is true')
  File "/usr/local/bin/cot", line 120, in tell
    return run_osascript(script, is_async)
  File "/usr/local/bin/cot", line 92, in run_osascript
    raise CalledProcessError(p.returncode, script)
subprocess.CalledProcessError: Command 'tell app "System Events" to name of first process whose frontmost is true' returned non-zero exit status 1
$ /usr/local/bin/cot .bashrc
Traceback (most recent call last):
  File "/usr/local/bin/cot", line 360, in <module>
    main(args, stdin)
  File "/usr/local/bin/cot", line 269, in main
    client_name = system.tell('name of first process whose frontmost is true')
  File "/usr/local/bin/cot", line 120, in tell
    return run_osascript(script, is_async)
  File "/usr/local/bin/cot", line 92, in run_osascript
    raise CalledProcessError(p.returncode, script)
subprocess.CalledProcessError: Command 'tell app "System Events" to name of first process whose frontmost is true' returned non-zero exit status 1
$ /Applications/CotEditor.app/Contents/SharedSupport/bin/cot .bashrc
Traceback (most recent call last):
  File "/Applications/CotEditor.app/Contents/SharedSupport/bin/cot", line 360, in <module>
    main(args, stdin)
  File "/Applications/CotEditor.app/Contents/SharedSupport/bin/cot", line 269, in main
    client_name = system.tell('name of first process whose frontmost is true')
  File "/Applications/CotEditor.app/Contents/SharedSupport/bin/cot", line 120, in tell
    return run_osascript(script, is_async)
  File "/Applications/CotEditor.app/Contents/SharedSupport/bin/cot", line 92, in run_osascript
    raise CalledProcessError(p.returncode, script)
subprocess.CalledProcessError: Command 'tell app "System Events" to name of first process whose frontmost is true' returned non-zero exit status 1
```


## Expected Result

editor window with .bashrc or new file

**Comment by elkssing** [comment]
Thanks!

My Automation panel showed Terminal with only System Events beneath, unchecked.  After I checked it and typed `cot .bashrc` into Terminal, the dialog you show above appeared.  When I clicked OK, a new CotEditor window opened with the contents of the file I wanted to edit.  Now the Automation panel looks like this:

<img width="780" alt="screen shot 2018-12-30 at 5 03 28 pm" src="https://user-images.githubusercontent.com/5316356/50551789-d7decd80-0c54-11e9-8611-2042032b15ad.png">

So I think adding documentation to the Help is a good idea and I think you already know how it should look.  😃

**Comment by 1024jp** [comment]
@elkssing Glad to hear the problem was solved. Furthermore, thank you for the feedback. Now, I'm clear about it. Besides the help, I've also improved the error message for that authorization issue instead of showing the strange stack trace.

<img width="682" alt="screen shot 2018-12-31 at 4 55 31 pm" src="https://user-images.githubusercontent.com/1165044/50556491-01fbc280-0d1d-11e9-8578-f55a27288d9e.png">
